features/media-regenerate.feature

```diff
@@ -383,3 +383,216 @@ Feature: Regenerate WordPress attachments
       Success: Regenerated 1 of 1 images.
       """
     And the wp-content/uploads/white-150-square-150x150.jpg file should not exist
+
```

Comment:
Added this explicitly (though `--only-missing` causes `--skip-delete` to be set anyway (https://github.com/wp-cli/media-command/blob/master/src/Media_Command.php#L78)) but when coming up with a test for this I noticed a bad bug I missed in the last PR https://github.com/wp-cli/media-command/pull/5: when `--only-missing` is set it doesn't take into account that the width/height/crop of an image size could have changed - a pretty crucial use case! It affects both running with and without `--image_size` but I've (hopefully) fixed it in the current commit though I can separate it out and open a new issue/PR if it's better for procedure.